src/psyclone/psyir/nodes/schedule.py

```diff
@@ -95,6 +96,12 @@ def gen_code(self, parent):
         for child in self.children:
             child.gen_code(parent)
 
+    def debug_string(self):
+        debug_string = ""
+        for statement in self.children:
+            debug_string += statement.debug_string()
+        return debug_string
+
```

Comment:
I don't think we need this (and associated test) now that a "schedule_node" visitor has been introduced to the Writer, do we?

Reply:
You're right, I'd forgotten this defaults to `DebugWriter` and thus to `FortranWriter`. They're now removed.